doc/source/users_guide/running-special-cases/Running-with-MOAR-data-as-atmospheric-forcing-to-spinup-the-model.rst

```diff
@@ -21,7 +21,7 @@ Example: Simulation with MOAR Data on cheyenne
    > ./xmlchange DATM_CPL_CASE=b40.1850.track1.1deg.006a
    # The following sets the align year and years to run over for atm forcing
    #  (you could also use an editor)
-   > ./xmlchange DATM_CPL_YR_ALIGN=1,DATM_CPL_YR_START=960,DATM_CPL_YR_END=1030
+   > ./xmlchange DATM_YR_ALIGN=1,DATM_YR_START=960,DATM_YR_END=1030
```

Comment:
Replaced in this file, but...
is MOAR still a thing or should I remove the file altogether?

Reply:
Good question, I don't know. However, if it is still a thing, `DATM_CPL_CASE` is also outdated. @ekluzek?

Reply:
@samsrabin thank you for pointing that out. Seems DATM_CPLHIST_CASE replaces DATM_CPL_CASE, so I'm changing it.

Reply:
I've added a warning box about this page being outdated, because the instructions still don't work, but it's not worth fixing them now.

Reply:
Thank you @samsrabin